README.md

```diff
@@ -160,7 +160,7 @@ Maven needs the user home to download artifacts to, and if the user does not exi
 
 For example, to run as user `1000` mounting the host' Maven repo
```

Comment:
thanks, if you change the command, the doc should also change

Reply:
Done.